src/frontend/src/pages/FinancePage/Finance.tsx

```diff
@@ -0,0 +1,17 @@
+/*
+ * This file is part of NER's FinishLine and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+import { Route, Switch } from 'react-router-dom';
+import { routes } from '../../utils/routes';
+import FinancePage from './FinancePage';
+
+const Finance: React.FC = () => {
+  return (
+    <Switch>
+      <Route path={routes.FINANCE} component={FinancePage} />
```

Comment:
are there any permissions things we should do with this

Reply:
we should check if they're on the finance team, best done in another ticket tho